pkgs/development/python-modules/torchrl/default.nix

```diff
@@ -0,0 +1,159 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, ninja
+, setuptools
+, wheel
+, which
+, cloudpickle
+, numpy
+, torch
+, ale-py
+, gym
+, pygame
+, torchsnapshot
+, gymnasium
+, mujoco
+, moviepy
+, git
+, hydra-core
+, tensorboard
+, tqdm
+, wandb
+, packaging
+, tensordict
+, flaky
+, imageio
+, pytest-xdist
+, pytestCheckHook
+, pyyaml
+, scipy
+}:
+
+buildPythonPackage rec {
+  pname = "torchrl";
+  version = "0.2.1";
+  # When using the pyproject format, the build fails because the pyproject.toml file contains "unknown properties":
+  # > ERROR Failed to validate `build-system` in pyproject.toml: Unknown properties: excludes, target-version, first_party_detection
```

Comment:
would be good to create an upstream issue for this, but not necessarily a blocker

Reply:
I have opened a PR upstream (https://github.com/pytorch/rl/pull/1828) and fetched my patch to get the `pyproject` install working.